gatsby-theme-untied/src/components/Navigation/__tests__/__snapshots__/navigation.test.js.snap

```diff
@@ -7,7 +7,17 @@ exports[`Navigation renders correctly - navigation bottom center 1`] = `
     id="nav-link"
     onClick={[Function]}
   >
-    ☰
+    <svg
```

Comment:
This SVGs are supposed to live in their own component to allow color settings via parameter.